debian/kafka-connect/Dockerfile

```diff
@@ -26,7 +26,7 @@ LABEL io.confluent.docker.build.number=$BUILD_NUMBER
 
 ENV COMPONENT=kafka-connect
 
-RUN echo "===> Installing JDBC, Elasticsearch and Hadoop connectors ..." \
+RUN echo "===> Installing JDBC, Elasticsearch, Hadoop, S3, and JMS connectors ..." \
```

Comment:
maybe say "installing connectors from apt" here and "installing connectors from confluent-hub" below?